yarn.lock

```diff
@@ -18236,7 +18225,7 @@ string-replace-loader@^3.1.0:
     loader-utils "^2.0.0"
     schema-utils "^3.0.0"
 
-"string-width-cjs@npm:string-width@^4.2.0", string-width@^4.1.0, string-width@^4.2.0, string-width@^4.2.3:
+"string-width-cjs@npm:string-width@^4.2.0":
```

Comment:
I keep seeing these same changes for `string-width`, `strip-ansi`, `wrap-ansi` whenever we update the package.json. I feel like it's been happening for months